pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -76,24 +77,17 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) error
 // PrintUpStreamVersions prints the origin versions which satisfies the following conditions:
 // 	1. Major versions greater than or equal to the minimum supported and default version
 //	2. Pre-release versions greater than default version
-func PrintUpStreamVersions(output io.Writer, minSupportedVersion string, defaultVersion string) error {
-	var releaseList []string
-	data, err := getGithubReleases()
+func PrintUpStreamVersions(output io.Writer, minSupportedVersion, defaultVersion string) error {
```

Comment:
@budhram  The only change I can see here is you have changed `minSupportedVersion string` to `minSupportedVersion`. Any specific reason for this?

Reply:
@LalatenduMohanty Will revert it, else this is the Go idiom :wink:

Reply:
I also do not see the need to be obscure. please leave code readable.
I do not care what 'Gophers' like... I care about readability and non-guessing.